### PR TITLE
Mocha (osu!taiko mascot) newspost announcement

### PR DESCRIPTION
**Pull request by Ephemeralis**
mocha mocha mocha mocha mocha mocha mocha mocha mocha mocha mocha mocha mocha mocha

### CODE REVIEW

File: news/2017-05-25-the-new-osutaiko-mascot-is-here.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+After almost a year in the making, the new osu!taiko mascot is finally ready to enter the scene. Say hello to **Mocha!**
+[[MORE]]
+
+<img src="https://assets.ppy.sh/media/mocha/mocha-header.jpg">
+
+Over ten months ago, [we held a contest](https://osu.ppy.sh/community/contests/2) to determine the visual design for the mascot of the osu!taiko gamemode. The contest descended into legend as we deliberated on the results - many of the community-made mascots rising into infamy through the various fan art contests we held in the meanwhile.
+
+It is time.
+
+We present to you, the brand new official mascot for osu!taiko - Mocha!<br><br><iframe src="https://docs.google.com/gview?url=https://assets.ppy.sh/media/mocha/Mocha_Documentation.pdf&embedded=true" style="width:750px; height:850px;" frameborder="0"></iframe>
```
Comment (sr229) [inline news/2017-05-25-the-new-osutaiko-mascot-is-here.md]: L18: unfortunately, the following HTML tags here doesn't work. Probably use Markdown embedding?

Comment (Ephemeralis) [inline news/2017-05-25-the-new-osutaiko-mascot-is-here.md]: markdown has no native pdf embedding, the only alternative would be to use object tags to embed the pdf and hope for browser support

Comment (sr229) [inline news/2017-05-25-the-new-osutaiko-mascot-is-here.md]: thanks for clarifying but can we do something like this ```html
[..] We present to you, the brand new official mascot for osu!taiko - Mocha!

<br/>
<br/>
<iframe src="https://docs.google.com/gview?url=https://assets.ppy.sh/media/mocha/Mocha_Documentation.pdf&embedded=true" style="width:750px; height:850px;" frameborder="0">
</iframe>
```

Comment (sr229) [inline news/2017-05-25-the-new-osutaiko-mascot-is-here.md]: it may work since GFM does render HTML

Comment (nyquillerium) [inline news/2017-05-25-the-new-osutaiko-mascot-is-here.md]: We've already discussed this in #osu-wiki iirc. News posts will be allowed HTML.

**Pull request merged**